src/main/java/view/ClubHomeView.java

```diff
@@ -13,11 +13,11 @@ public class ClubHomeView extends JPanel {
     private JPanel center;
     private JLabel idk;
 
-    private final String viewName = "club home";
-    private final ClubHomeViewModel clubHomeViewModel;
+    private final String viewName = "logged in";
```

Comment:
viewName should be changed to "club logged in" to differentiate from student logged in view.

Reply:
Changed. Thanks!